drivers/unix/net_socket_unix.h

```diff
@@ -28,25 +28,18 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
-#ifndef NET_SOCKET_POSIX_H
-#define NET_SOCKET_POSIX_H
+#ifndef NET_SOCKET_UNIX_H
+#define NET_SOCKET_UNIX_H
 
-#include "core/io/net_socket.h"
+#if defined(UNIX_ENABLED) && !defined(UNIX_SOCKET_UNAVAILABLE)
 
-#if defined(WINDOWS_ENABLED)
-#include <winsock2.h>
-#include <ws2tcpip.h>
-#define SOCKET_TYPE SOCKET
+#include "core/io/net_socket.h"
 
-#else
 #include <sys/socket.h>
-#define SOCKET_TYPE int
-
-#endif
 
 class NetSocketPosix : public NetSocket {
 private:
-	SOCKET_TYPE _sock; // NOLINT - the default value is defined in the .cpp
+	int _sock; // NOLINT - the default value is defined in the .cpp
```

Comment:
// NOLINT is probably no longer necessary as SOCKET_TYPE is now no longer a macro